hooks/pre/35-change-evr-extension-ownership.rb

```diff
@@ -0,0 +1,26 @@
+# In Katello 4.16, the 'evr' extension is removed from PostgreSQL and integrated into the Katello database via a migration.
+# This hook ensures the 'evr' extension's ownership is transferred to the 'foreman' user so migrations can act on it.
+
+if local_postgresql? && execute("rpm -q postgresql-evr", false, false)
```

Comment:
Probably better to specifically check that Foreman is local instead of there being any local DB:
```suggestion
if local_foreman_db? && execute("rpm -q postgresql-evr", false, false)
```

Though perhaps you can also include `devel_scenario?` in there if you want to accommodate for the katello-devel boxes.

Reply:
Updated to 
```
if (local_foreman_db? || devel_scenario?) && execute("rpm -q postgresql-evr", false, false)
```